mne_bids_pipeline/_run.py

```diff
@@ -277,14 +279,29 @@ def clear(self) -> None:
         self.memory.clear()
 
 
-def save_logs(*, config: SimpleNamespace, logs) -> None:  # TODO add type
+def save_logs(*, config: SimpleNamespace, logs: list[pd.Series]) -> None:
     fname = config.deriv_root / f"task-{get_task(config)}_log.xlsx"
 
     # Get the script from which the function is called for logging
     sheet_name = _short_step_path(_get_step_path()).replace("/", "-")
     sheet_name = sheet_name[-30:]  # shorten due to limit of excel format
 
-    df = pd.DataFrame(logs)
+    # We need to make the logs more compact to be able to write Excel format
+    # (32767 char limit per cell), in particular the "cfg" column has very large
+    # cells, so replace the "cfg" column with a "cfg_zip" column (parentheticals
+    # contain real-world example numbers used to illustrate the compression):
```

Comment:
@hoechenberger better?

Reply:
it helps my understanding, thanks! but does that mean that this column isn't human-readable anymore?

Reply:
... at least that's the case for the ERP_CORE dataset I was looking at, which is probably a worst case. There are probably some steps for which compression is not necessary. But it's hard to know which will be which.

I think a big part of it might have been the support for a custom Montage -- the `json_tricks` dump of that was super long.

Reply:
Could we, perhaps, create a new sheet and write one line per row (easy) or one config setting per row (more difficult), instead? this would surely keep us below the character limit per cell and retain readability

Reply:
This was actually fairly easy to implement so I tried it, this is the result:

![Screenshot from 2024-02-29 16-09-09](https://github.com/mne-tools/mne-bids-pipeline/assets/2365790/0585e5b2-ef84-4801-bbf3-f70fa77f49ff)

I'll push a commit in case it's a reasonable way to go.

Reply:
Thanks for looking into this

I'm wondering if we need this level of "differentiability" across participants, sessions, tasks, and steps.

I claim that currently, the only really reliable and "sound" way to use the pipeline is by processing all data with the same configuration. If different settings are supposed to be used for different sessions or tasks, a new, separate derivatives folder should be used.

If we follow this premise, we could create a single Excel sheet where one row corresponds to one line or one setting in the config file that was used

WDYT?